apps/ensnode/src/lib/label-helpers.ts

```diff
@@ -0,0 +1,53 @@
+import { isLabelIndexable } from "ensnode-utils/subname-helpers";
+import type { Labelhash } from "ensnode-utils/types";
+import { EnsRainbowApiClient } from "ensrainbow-sdk/client";
+import { DEFAULT_ENSRAINBOW_URL } from "ensrainbow-sdk/consts";
+import { labelHashToBytes } from "ensrainbow-sdk/label-utils";
+import { normalize } from "viem/ens";
+
+const ensRainbowApiClient = new EnsRainbowApiClient({
+  endpointUrl: new URL(process.env.ENSRAINBOW_URL || DEFAULT_ENSRAINBOW_URL),
+});
+
+/**
+ * Heal a labelhash to its original label.
+ *
+ * @returns a healed label for a given labelhash, if possible.
+ *
+ * NOTE: undefined return type is helpful for passing to drizzle, which
+ * will ignore the input rather that set the value in the db to `NULL`
+ * if we were to return `null`.
+ **/
+export async function heal(labelhash: Labelhash) {
```

Comment:
```suggestion
export async function labelByHash(labelhash: Labelhash) {
```

Suggesting this renaming to better align with ENS Subgraph code that uses `nameByHash` (except fixing their misuse of the word "name").